README.md

```diff
@@ -43,13 +43,13 @@ pak::pkg_install(pkg_list)
 
 ### Use `renv` to recreate our project library
 
-Another option is to use the [`renv` package](https://rstudio.github.io/renv/index.html) to replicate our exact project library. `renv` will create a private, project-specific library that is separate from your personal library of packages. This would be a good option if, for example, you have another project that relies on a specific version of a package and you don't want to mess with it by upgrading, downgrading, etc.
+Another option is to use the [`renv` package](https://rstudio.github.io/renv/index.html) to replicate our exact project library. `renv` is a package that uses a [snapshot and restore](https://environments.rstudio.com/snapshot.html) strategy to create **r**eproducible **env**nvironments for R projects. `renv` will create a private, project-specific library that is separate from your personal library of packages. This would be a good option if, for example, you have another project that relies on a specific version of a package and you don't want to mess with it by upgrading, downgrading, etc.
 
-If you want to know more about what `renv` is doing behind the scenes see [Introduction to renv](https://rstudio.github.io/renv/articles/renv.html) and [Snapshot and Restore](https://environments.rstudio.com/snapshot.html#pre-requisite-steps).
+If you would want to learn more about what `renv` is doing behind the scenes and how you can use `renv` for collaboration see [Introduction to renv](https://rstudio.github.io/renv/articles/renv.html), [Collaborating with renv](https://rstudio.github.io/renv/articles/collaborating.html), and [Snapshot and Restore](https://environments.rstudio.com/snapshot.html#pre-requisite-steps).
 
```

Comment:
I suggest linking only to renv's main site, Or at least only include more detailed links when they are mission critical. URLs have a way of changing 🙂